hs/app/reach/Main.hs

```diff
@@ -2451,32 +2451,31 @@ whoami = command "whoami" $ info f fullDesc
   where
     f = pure . script $ write [N.text| echo "$whoami'" |]
 
-type DeviceCode = Text
-
 -- https://stackoverflow.com/a/60790430
-newtype SomeValue = SomeValue String
-instance FromJSON SomeValue where
-  parseJSON = withObject "SomeValue" $ \o -> SomeValue <$> o .: "html_url"
+newtype GitHubGistResponse = GitHubGistResponse String
+instance FromJSON GitHubGistResponse where
+  parseJSON = withObject "GitHubGistResponse" $ \o -> GitHubGistResponse <$> o .: "html_url"
 
 support :: Subcommand
 support = command "support" $ info (pure step1) d
   where
     d = progDesc "Upload index.rsh and index.mjs to help us troubleshoot!"
-
     splitByAmpersands :: BSLC8.ByteString -> [Text]
     splitByAmpersands s = T.splitOn "&" (pack $ BSLC8.unpack s)
 
```

Comment:
Formatting!

Reply:
Oh! You want me to remove all the blank lines.

Reply:
I believe [3c89cc9](https://github.com/reach-sh/reach-lang/pull/762/commits/3c89cc91ff31c1edb98db13116ecb01a23d6ff4e) should at least partially address this.